ui/app/routes/vault/cluster/oidc-callback.js

```diff
@@ -6,28 +6,17 @@ export default Route.extend({
     // left blank so we render the template immediately
   },
   afterModel() {
-    const queryString = decodeURIComponent(window.location.search);
-    // Since state param can also contain namespace, fetch the values using native url api.
-    // For instance, state params value can be state=st_123456,ns=d4fq
-    // Ember paramsFor used to strip out the value after the "=" sign. In short ns value was not being passed along.
```

Comment:
Not sure if we should still account for this, it didn't seem to be an issue (so perhaps is fixed?) But thinking it may be reasonable to add a check and compare the state of the queryString against the state from `this.paramsFor`